staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2683,8 +2683,19 @@ type InterfaceBindingPlugin struct {
 	// If namespace is not specified, VMI namespace is assumed.
 	// version: 1alphav1
 	NetworkAttachmentDefinition string `json:"networkAttachmentDefinition,omitempty"`
+	// DomainAttachment is a standard domain definition that kubevirt supports.
+	// Currently, only "Tap" is supported.
```

Comment:
I would prepare it for the future and phrase it as `Supported values: "tap".`.

Reply:
done.